Undefined                       first referenced
 symbol                             in file
_ZNK6SkPath4dumpEP9SkWStreamb       lib/libSkia.a(SkPathOpsDebug.cpp.o)

--- webkitgtk-2.52.0/Source/ThirdParty/skia/src/pathops/SkPathOpsDebug.cpp.old
+++ webkitgtk-2.52.0/Source/ThirdParty/skia/src/pathops/SkPathOpsDebug.cpp
@@ -2895,7 +2895,6 @@
 
 static void dump_path(FILE* file, const SkPath& path, bool dumpAsHex) {
     SkDynamicMemoryWStream wStream;
-    path.dump(&wStream, dumpAsHex);
     sk_sp<SkData> data(wStream.detachAsData());
     fprintf(file, "%.*s\n", (int) data->size(), (char*) data->data());
 }
@@ -3091,9 +3090,9 @@
 
 // global path dumps for msvs Visual Studio 17 to use from Immediate Window
 void Dump(const SkPath& path) {
-    path.dump();
+  fprintf(stderr, "Skipping path.dump\n");
 }
 
 void DumpHex(const SkPath& path) {
-    path.dumpHex();
+  fprintf(stderr, "Skipping path.dumphex\n");
 }
